x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/persistence/MockClientBuilder.java

```diff
@@ -77,14 +56,11 @@
     private ClusterAdminClient clusterAdminClient;
     private IndicesAdminClient indicesAdminClient;
 
-    private IndicesAliasesRequestBuilder aliasesRequestBuilder;
```

Comment:
In hindsight I'm not sure this mock client class was useful. I remove all unused methods.